ganttproject/src/net/sourceforge/ganttproject/export/ExporterToExcel.java

```diff
@@ -0,0 +1,54 @@
+package net.sourceforge.ganttproject.export;
+
+import biz.ganttproject.core.option.GPOptionGroup;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+
+import java.awt.*;
```

Comment:
You need just a single `Component` from the whole `awt`, it makes sense to remove wildcard import